src/VisualStudio/Core/Def/Implementation/AbstractVsTextViewFilter`2.cs

```diff
@@ -32,7 +33,20 @@ internal abstract class AbstractVsTextViewFilter<TPackage, TLanguageService> : A
             _languageService = languageService;
         }
 
-        public virtual int GetDataTipText(TextSpan[] pSpan, out string pbstrText)
+        int IVsTextViewFilter.GetDataTipText(TextSpan[] pSpan, out string pbstrText)
+        {
+            try
+            {
+                return GetDataTipTextImpl(pSpan, out pbstrText);
+            }
+            catch (Exception e) when (FatalError.ReportWithoutCrash(e))
+            {
+                pbstrText = null;
```

Comment:
I'd consider rethrowing, so that way we're still converting to the right HRESULT, IErrorInfo is correctly set, etc.